src/database/schemas/address.schema.ts

```diff
@@ -0,0 +1,59 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { Document } from 'mongoose';
+import { CountryEnum } from '../enums/country.enum';
+
+export type AddressType = Address & Document;
+
+@Schema({ _id: false })
+export class Address {
+    @Prop({
+        type: String,
+        trim: true,
+        lowercase: true,
+        maxLength: 256,
+    })
+    street: string;
+
+    @Prop({
+        type: String,
+        trim: true,
+        lowercase: true,
+        maxLength: 8,
+    })
+    streetNumber: string;
+
+    @Prop({
+        type: String,
+        trim: true,
+        lowercase: true,
+        maxLength: 256,
+    })
+    city: string;
+
+    @Prop({
+        type: String,
+        trim: true,
+        lowercase: true,
+        maxLength: 64,
+    })
+    state: string;
+
+    @Prop({
+        type: String,
+        trim: true,
+        lowercase: true,
+        maxLength: 16,
```

Comment:
issue(blocking): This is way too long. Are there any zip codes that long?

Reply:
Longest zipcode is 10 digits. Again, should we stop with the potencies?